### PR TITLE
Cooking Oil Recipe 

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

Cooking oil can now be made with:
1 Part Oil
1 Part Sugar
1 Part Hydrogen
1 Part Carbon

## Why It's Good For The Game

The chef starts with a 1000u vat of cooking oil at the start of the shift, however if said vat is damaged or destroyed or if you do not have access to it because you are building a shuttle in space or are a lavaland golem or something you wont be able to deep fry anything even if you make the machine because you lack cooking oil.

## Changelog
:cl:
add: Cooking oil can now be made with 1 part oil, 1 part sugar, 1 part hydrogen and 1 part carbon.
/:cl:

### CODE REVIEW

**Comment by Isy232** [comment]
Tested it locally, it works fine.

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -7,6 +7,12 @@
 	results = list(/datum/reagent/space_cleaner/sterilizine = 3)
 	required_reagents = list(/datum/reagent/consumable/ethanol = 1, /datum/reagent/medicine/charcoal = 1, /datum/reagent/chlorine = 1)
 
+/datum/chemical_reaction/cooking_oil
+	name = "Spray Tan"
```
Comment (Dingo-Dongler) [inline code/modules/reagents/chemistry/recipes/others.dm]: Shouldn't this be name = "Cooking Oil"

Comment (Isy232) [reply code/modules/reagents/chemistry/recipes/others.dm]: Yes, it should, I forgot that bit...

Comment (Isy232) [reply code/modules/reagents/chemistry/recipes/others.dm]: Done

Comment (Isy232) [reply code/modules/reagents/chemistry/recipes/others.dm]: Thanks

**Pull request merged**